pkg/api/handlers/tasks.go

```diff
@@ -704,6 +704,10 @@ func (t *TaskHandler) CreateFromScope(req api.Context) error {
 		return fmt.Errorf("failed to generate alias: %w", err)
 	}
 
+	if len(workflowManifest.Alias) > 12 {
```

Comment:
I can only shorten the alias for email receiver, but let me know if this is going to cause problems. Making it consistent for other things.